src/schema/v2/__tests__/ordered_sets.test.js

```diff
@@ -19,15 +19,17 @@ xdescribe("OrderedSets type", () => {
 
   const context = {
     setsLoader: sinon.stub().returns(
-      Promise.resolve([
-        {
-          id: "52dd3c2e4b8480091700027f",
-          item_type: "Gene",
-          key: "artists:featured-genes",
-          name: "Featured Genes",
-          description: "These Genes are featured",
-        },
-      ])
+      Promise.resolve({
+        body: [
+          {
+            id: "52dd3c2e4b8480091700027f",
+            item_type: "Gene",
+            key: "artists:featured-genes",
+            name: "Featured Genes",
+            description: "These Genes are featured",
+          },
+        ],
+      })
```

Comment:
This spec doesn't run because v2 doesn't have this exposed (yet).

Reply:
Oh, gotcha, the root field is commented out.